specs/core/0_beacon-chain.md

```diff
@@ -239,17 +239,17 @@ The following values are (non-configurable) constants used throughout the specif
 | `MAX_DEPOSITS` | `2**4` (= 16) |
 | `MAX_VOLUNTARY_EXITS` | `2**4` (= 16) |
 
-### Domain types
+### Tag types
 
-The following types are defined, mapping into `DomainType` (little endian):
+The following types are defined, mapping into `TagType` (little endian):
 
 | Name | Value |
 | - | - |
-| `DOMAIN_BEACON_PROPOSER` | `0` |
-| `DOMAIN_BEACON_ATTESTER` | `1` |
-| `DOMAIN_RANDAO` | `2` |
-| `DOMAIN_DEPOSIT` | `3` |
-| `DOMAIN_VOLUNTARY_EXIT` | `4` |
+| `TAG_BEACON_PROPOSER` | `0` |
```

Comment:
I don't like the inconsistency in how these are presented here as numbers "mapping into bytes" as opposed to their presentation as literal bytestrings in the config files. 

Is there any reason to not display the literal bytestrings here as well?

Reply:
It is a function of how pyspec-SSZ works. It needs a bytes input, and python bytes look ugly.

However, there is a special case already in the pyspec builder for handling Tags, so I have modified that to work with raw bytes. Have a look at the update and let me know what you think of it. :)

Reply:
I like it!